odxtools/parameters/systemparameter.py

```diff
@@ -1,19 +1,25 @@
 # SPDX-License-Identifier: MIT
+import getpass
 from dataclasses import dataclass
+from datetime import datetime
 from typing import List, Optional
 from xml.etree import ElementTree
 
 from typing_extensions import override
 
-from ..decodestate import DecodeState
 from ..encodestate import EncodeState
-from ..exceptions import odxrequire
+from ..exceptions import odxraise, odxrequire
 from ..odxlink import OdxDocFragment
 from ..odxtypes import ParameterValue
 from ..utils import dataclass_fields_asdict
 from .parameter import ParameterType
 from .parameterwithdop import ParameterWithDOP
 
+PREDEFINED_SYSPARAM_VALUES = [
```

Comment:
Those changes should not be in this PR

Reply:
they are not really: since the changes to `parameter_info()` require the changes of #347, the changes this PR is build on top of #347, i.e., this stuff will go away once #347 has been merged...